demo/index.html

```diff
@@ -27,105 +30,136 @@
         </li>
         <li class="c-tab__list__item" tabindex="0">
-          <a href="//zendeskgarden.github.io/css-components/" tabindex="-1">CSS Components</a>
+          <a href="//zendeskgarden.github.io/css-components" tabindex="-1">CSS Components</a>
         </li>
         <li class="c-tab__list__item is-selected" tabindex="0">
-          <a href="/react-components" tabindex="-1">React Components</a>
+          <a href="//zendeskgarden.github.io/react-components/" tabindex="-1">React Components</a>
         </li>
       </ul>
     </div>
   </nav>
+  <header class="c-header c-main__header">
+    <figure class="c-header__logo">
+      <a class="u-fg-inherit" href="/">
+        <svg class="c-header__logo__svg svg-square">
+          <use xlink:href="index.svg#zd-svg-icon-26-zendesk">
+        </svg>
+      </a
+    ></figure
+    >
+    <ul class="c-ctl c-ctl--header">
+      <li class="c-ctl__item u-position-relative">
+        <a class="c-menu-ctl c-header__menu js-menu" href="#settings">
+          &nbsp;
+          <svg>
+            <use xlink:href="index.svg#zd-svg-icon-16-menu-stroke">
+          </svg>
+        </a>
+        <ul aria-hidden="true" class="c-menu c-menu--down c-menu--sm" role="menu">
+          <a href="/" class="c-menu__item" role="menuitem">
+            Home
+          </a>
+          <a href="/assets" class="c-menu__item" role="menuitem">
+            Assets
+          </a>
+          <a href="//zendeskgarden.github.io/css-components" class="c-menu__item" role="menuitem">
+            CSS Components
+          </a>
+          <a href="//zendeskgarden.github.io/react-components" class="c-menu__item" role="menuitem">
+            React Components
+          </a>
+        </ul>
+      </li>
+    </ul>
+  </header>
   <main class="c-main">
-    <h1 class="c-main__title">React Components</h1>
-    <p class="c-main__subtitle u-delta u-mt-lg" style="white-space: normal;">
-      This documentation includes React components and utilities that provide visuals, keyboard-navigation, localization,
-      and accessibility defined within the Garden Design System.
-    </p>
-    <div class="c-main__body" style="margin-top: 0; width: 720px">
-      <div class="l-grid l-grid--xxl u-beta">
-        <div class="l-grid__item u-1/4">
-          <div class="u-mb">
+    <div class="container-fluid">
+      <h1 class="c-main__title">React Components</h1>
+      <p class="c-main__subtitle u-delta u-mt-lg" style="white-space: normal;">
+        This documentation includes React components and utilities that provide visuals, keyboard-navigation, localization,
+        and accessibility defined within the Garden Design System.
+      </p>
+      <div class="c-main__body" style="margin-top: 0; margin-left: 16px; margin-right: 16px;">
+        <div class="row">
+          <div class="col-xl-3 col-md-6 u-mb">
             <a class="u-fg-inherit" href="avatars">Avatars</a>
             <p class="u-zeta">@zendeskgarden/react-avatars</p>
```

Comment:
Remove all of these – they are muddying up this index page.

Reply:
Removed.